content/en/examples/controllers/daemonset.yaml

```diff
@@ -35,6 +35,10 @@ spec:
         volumeMounts:
         - name: varlog
           mountPath: /var/log
+      # system-node-critical is the highest priority class and can be used to ensure that
+      # a critical daemonset's pod will evict an existing pod if that is required to allow
```

Comment:
```suggestion
      # a node-critical pod will preempt an existing Pod if that is required to allow
```

Reply:
I reworked the language a bit, I didn't like the comment being about `system-node-critical`, with the commented out example being `system-cluster-critical`.  I made it a bit more generic and just used `system-cluster-critical` in the example. WDYT?

Reply:
On reflection, even that's higher that I'm really happy with.